crates/meroctl/src/cli/context/invite.rs

```diff
@@ -74,31 +71,45 @@ impl InviteCommand {
     }
 
     pub async fn invite(&self, environment: &Environment) -> EyreResult<ContextInvitationPayload> {
-        let config = load_config(&environment.args.home, &environment.args.node_name).await?;
+        let connection = environment
+            .connection
+            .as_ref()
+            .ok_or_eyre("No connection configured")?;
 
-        let multiaddr = fetch_multiaddr(&config)?;
+        let auth_key = connection
+            .auth_key
+            .as_ref()
+            .ok_or_eyre("No authentication key configured")?;
```

Comment:
an authentication key is not "required"

Reply:
I see you marked the rest of the review as resolved. Is this the only change requested?

Reply:
Yep, looks to me like this is the last thing.